site/docs/guides/spark-s3.md

```diff
@@ -16,7 +16,7 @@ docker run -p 19120:19120 ghcr.io/projectnessie/nessie:latest
 Note: this example will run the Nessie Server using in-memory storage for table metadata. If/when the container
 is deleted, Nessie data about table changes will be lost, yet the data files in S3 will remain.
 
-## Setting up Spark Session
+## Setting up Spark Session for Amazon S3
 
 Configure an AWS profile (e.g. called `demo`) in `~/.aws/credentials` (or other location appropriate for your OS)
```

Comment:
1. Missing period
2. Maybe link users to https://docs.aws.amazon.com/cli/latest/userguide/cli-configure-files.html ? It wasn't obvious for me how to create that file.

Reply:
rewrote this paragraph